README.md

```diff
@@ -1,17 +1,121 @@
-# Cluster Ingress Operator
+# OpenShift Ingress Operator
 
-Cluster Ingress Operator deploys and manages [HAProxy](http://www.haproxy.org) to provide highly available network ingress in [OpenShift](https://openshift.io/), enabling both OpenShift [Routes](https://docs.okd.io/latest/rest_api/apis-route.openshift.io/v1.Route.html) and Kubernetes [Ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/).
 
-The operator tries to be useful out of the box by creating a working default deployment based on the cluster's configuration.
+#### Ingress Operator enables Routes and Kubernetes Ingress on OpenShift.
 
-* On cloud platforms, ingress is exposed externally using Kubernetes [LoadBalancer Services](https://kubernetes.io/docs/concepts/services-networking/#loadbalancer).
-* On Amazon AWS, the operator publishes a wildcard DNS record based on the [cluster ingress domain](https://github.com/openshift/api/blob/master/config/v1/types_ingress.go) and pointing to the service's load-balancer.
+Ingress Operator enables external traffic to reach the services that expose
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) or
+[Route](https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html)
+resources.
 
-## How to help
+To provide this functionality, Ingress Operator deploys and manages the
+[OpenShift router](https://github.com/openshift/router) — a
```

Comment:
"a"→"an"

Reply:
Fixed

Reply:
The name of the operator repo is "cluster-ingress-operator". The title of the doc is "OpenShift Ingress Operator". `ClusterIngress` has now transitioned to `IngressController`. OpenShift includes CR's for `cluster`/`ingresses.config.openshift.io` and supports the Kubernetes `Ingress` resource. I can see how a user would get confused. Should the name of the repo and readme title be more closely align to `IngressController`? For example, repo name "ingress-controller-operator" and title "OpenShift Ingress Controller Operator"?

Reply:
Changing the name of the repository is a more onerous endeavor and goes beyond the scope of this PR IMO.  Otherwise, the naming makes sense to me: the ingress operator manages the ingress controller, which works with the ingress (sort of) and (really) route resources.

Reply:
Ingress Controller Operator vs. Ingress Operator sounds like a good list discussion.

Changing the labels, annotations, and resource names would have a few considerations, mostly self-contained to this project, although we have to be careful about resource names (which I believe are currently being used as an informal contract in some cases and which could potentially be replaced by stable label selectors).

Changing the repo name would have enough ripple effects to warrant its own design document (this isn't meant to be snarky — just a statement of fact for consideration).